src/Microsoft.DotNet.XUnitExtensions/tests/ActiveIssueTests.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.DotNet.XUnitExtensions.Tests
+{
+    [TestCaseOrderer("Microsoft.DotNet.XUnitExtensions.Tests.AlphabeticalOrderer", "Microsoft.DotNet.XUnitExtensions.Tests")]
+    public class ActiveIssueTests
+    {
+        [Fact]
+        [ActiveIssue("https://github.com/repo/123")]
+        public void CheckIfSkipsFact()
+        {
+            Assert.False(true, "Should have been skipped.");
+        }
+
+        [Fact]
```

Comment:
Should probably be ConditionalFact given the test name
```suggestion
        [ConditionalFact]
```